docs/USER.md

```diff
@@ -270,6 +271,26 @@ Please note that SPI service didn't perform any specific validity checks
 for the user provided credentials, so it's assumed user provides a correct data.
 
 
+## Overriding default binding lifetime
+In general, bindings (and their dependant secrets) are not supposed to be long-lived. Their TTL in configurable globally, 
+and by default it is set to 2 hrs. But some tasks require more long-lived credentials to be available, so the creator of the binding
+may override the default setting by specifying `spec.lifetime` field of the binding.
+It is accepts any standard time periods, like `2h30m`, `90s` etc, except the negative values and values less than
```

Comment:
Any standard time period greater than `60s`, like `2h30m` or `90s`, is accepted. To express infinite lifetime, the value of `-1` is also allowed. The bindings with infinite lifetime will only be removed when deleted from the cluster.